lib/preface_make/bounded_meet_semilattice.mli

```diff
@@ -0,0 +1,54 @@
+(** Building a {!module:Preface_specs.Bounded_meet_semilattice} *)
+
+(** {1 Using the minimal definition} *)
+
+(** {2 Using meet and top}
+
+    Build a {!module-type:Preface_specs.BOUNDED_MEET_SEMILATTICE} using
+    {!module-type:Preface_specs.BOUNDED_MEET_SEMILATTICE.WITH_MEET_AND_TOP}. *)
+
+module Via_meet_and_top
+    (Req : Preface_specs.Bounded_meet_semilattice.WITH_MEET_AND_TOP) :
+  Preface_specs.BOUNDED_MEET_SEMILATTICE with type t = Req.t
+
+(** {1 Over Meet_semilattice functor}
```

Comment:
```diff
-  (** {1 Over Meet_semilattice functor}
+ (** {1 Over Meet_semilattice}
```

(I think that `Divisible` refers a `Contravariant functor`, but here, a semilattice is not a functor)